BO4E/BO/Auftrag.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Text.Json.Serialization;
+using BO4E.COM;
+using BO4E.ENUM;
+using BO4E.meta;
+using Newtonsoft.Json;
+using ProtoBuf;
+
+namespace BO4E.BO
+{
+    /// <summary>
+    /// Ein Auftrag beschreibt einen Vorgang, der von einem anderen Marktpartner auszuführen ist. 
+    /// </summary>
+    [NonOfficial(NonOfficialCategory.REGULATORY_REQUIREMENTS)]
+    [ProtoContract]
+    public abstract class Auftrag : BusinessObject
+    {
+        /// <summary>
+        /// workaround
+        /// </summary>
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(4, Name = nameof(Ausfuehrungsdatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        protected DateTime _Ausfuehrungsdatum
+        {
+            get => Ausfuehrungsdatum.UtcDateTime;
+            set => Ausfuehrungsdatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+
+        /// <summary>
+        /// Das Ausführungsdatum beschreibt zu welchem Zeitpunkt ein Auftrag ausgeführt werden soll.
+        /// </summary>
+        [JsonProperty("ausfuehrungsdatum", Required = Required.Always)]
+        [JsonPropertyName("ausfuehrungsdatum")]
+        [ProtoIgnore]
+        public DateTimeOffset Ausfuehrungsdatum { get; set; }
+
+        /// <summary>
+        /// workaround
+        /// </summary>
+        [System.Text.Json.Serialization.JsonIgnore]
+        [Newtonsoft.Json.JsonIgnore]
+        [ProtoMember(5, Name = nameof(Fertigstellungsdatum))]
+        [CompatibilityLevel(CompatibilityLevel.Level240)]
+        protected DateTime _Fertigstellungsdatum
+        {
+            get => Fertigstellungsdatum.UtcDateTime;
+            set => Fertigstellungsdatum = DateTime.SpecifyKind(value, DateTimeKind.Utc);
+        }
+
+        /// <summary>
+        /// Das Fertigstellungsdatum beschreibt zu welchem Zeitpunkt ein Auftrag ausgeführt wurde/wird.
+        /// </summary>
+        [JsonProperty("fertigstellungsdatum", Required = Required.Default)]
+        [JsonPropertyName("fertigstellungsdatum")]
+        [ProtoIgnore]
+        public DateTimeOffset Fertigstellungsdatum { get; set; }
+
+        /// <summary>
+        /// Die Adresse, die sich in Belieferung befindet.
+        /// </summary>
+        [JsonProperty("lieferanschrift", Required = Required.Default)]
+        [JsonPropertyName("lieferanschrift")]
+        [ProtoMember(6)]
+        public Adresse Lieferanschrift { get; set; }
+
+        /// <summary>
+        /// Die ID der Marktlokation der der zu sperrende Zähler zugeordnet ist.
+        /// </summary>
+        [JsonProperty("marktlokationsId", Required = Required.Always)]
+        [JsonPropertyName("marktlokationsId")]
+        [ProtoMember(7)]
+        public string MarktlokationsId { get; set; }
```

Comment:
Hängt der an der Malo? Hätte erwartet, dass das auf der Melo hängt?

Reply:
![grafik](https://user-images.githubusercontent.com/23094997/143416012-ac90fe0f-a3ed-469b-976e-5907e27e712e.png)
in der 17115/6/7 ist es die malo